docs/porting-addons-to-v2.md

```diff
@@ -215,9 +215,115 @@ Now that we've separated the test-app and docs app concerns from the addon, we c
-7. Grab the [example rollup config](https://github.com/embroider-build/embroider/blob/main/packages/addon-dev/sample-rollup.config.js) and save it as `addon/rollup.config.js`.
+6. Grab the example babel config (below) and save it as `addon/babel.config.json`
+   - If you addon requires template transforms in order to publish to a shareable format. Apply transforms using the `babel-plugin-ember-template-compilation`. View how to use this in the example babel.config.cjs (below).
+
+  <details><summary>example babel.config.json</summary>
+  
+
+  ```json
+  {
+    "plugins": [
+      "@embroider/addon-dev/template-colocation-plugin",
+      ["@babel/plugin-proposal-decorators", { "legacy": true }],
+      "@babel/plugin-transform-class-properties"
+    ]
+  }
+  ```
+
+  </details>
+
+  <details><summary>example babel.config.cjs</summary>
+
+  ```js
+  // Some addons need to transform their templates before they have a portable format.
+  // In "classic" builds this was done at the application. In embroider it should
+  // be done during the addon build.
+  const someAstTransformPlugin = require('./some-ast-transform-plugin');
+
+  module.exports = {
+    plugins: [
+      '@embroider/addon-dev/template-colocation-plugin',
+      [
+        'babel-plugin-ember-template-compilation',
+        {
+          targetFormat: 'hbs',
+          compilerPath: 'ember-source/dist/ember-template-compiler',
+          transforms: [
+            someAstTransformPlugin,
+            './path/to/another-template-transform-plugin',
+          ],
+        },
+      ],
+      ['@babel/plugin-proposal-decorators', { legacy: true }],
+      '@babel/plugin-transform-class-properties',
+    ],
+  };
+  ```
+
+  </details>
+
+
+7. Grab the this example rollup config and save it as `addon/rollup.config.mjs`.
+
+  <details><summary>example rollup.config.mjs</summary>
+
+  ```js
+  import babel from '@rollup/plugin-babel';
+  import { Addon } from '@embroider/addon-dev/rollup';
+
+  const addon = new Addon({
+    srcDir: 'src',
+    destDir: 'dist',
+  });
+
+  export default {
+    // This provides defaults that work well alongside `publicEntrypoints` below.
+    // You can augment this if you need to.
+    output: addon.output(),
+
+    plugins: [
+      // These are the modules that users should be able to import from your
+      // addon. Anything not listed here may get optimized away.
+      addon.publicEntrypoints(['components/**/*.js', 'index.js']),
+
+      // These are the modules that should get reexported into the traditional
+      // "app" tree. Things in here should also be in publicEntrypoints above, but
+      // not everything in publicEntrypoints necessarily needs to go here.
+      addon.appReexports(['components/welcome-page.js']),
```

Comment:
Should this be replaced with
```suggestion
      addon.appReexports([
      'components/**/*.js',
      'helpers/**/*.js',
      'services/**/*.js',
    ]),
```
to have something more generic? That's what's currently in the addon blueprint.

Reply:
yeah, this is good suggestion, thanks!